WireNetwork/Sources/WireNetwork/APIs/UpdateEventsAPI/UpdateEventsAPIV8.swift

```diff
@@ -16,6 +16,26 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
+import Foundation
+
 final class UpdateEventsAPIV8: UpdateEventsAPIV7 {
     override var apiVersion: APIVersion { .v8 }
+
+    override func getServerTime() async throws -> Date {
+        let path = "\(pathPrefix)/time"
+
+        var requestBuilder = try URLRequestBuilder(path: path).withMethod(.get)
+
+        let request = requestBuilder.build()
```

Comment:
suggestion: keep it simple:

```suggestion
        let request = try URLRequestBuilder(path: "\(pathPrefix)/time")
            .withMethod(.get)
            .build()
```